ocaml/idl/datamodel_host.ml

```diff
@@ -1459,6 +1459,18 @@ let host_query_ha = call ~flags:[`Session]
     ~allowed_roles:_R_POOL_ADMIN
     ()
 
+  let copy_primary_host_certs = call
+    ~name:"copy_primary_host_certs"
+    ~in_oss_since:None
+    ~in_product_since:rel_next
+    ~doc:"useful for secondary slaves that are missing some certs"
```

Comment:
I think this could be confusing as there are no primary slaves. It's secondary hosts or slaves.

Reply:
If we want to avoid the master/slave terminology then we should use just "secondary host" here, and considering you use 'primary' elsewhere it'd make more sense to drop the 'slave' terminology.